client/src/main/java/com/distkv/client/commandlinetool/CommandExecutorHandler.java

```diff
@@ -504,4 +505,76 @@ public static String intIncr(
     }
     return STATUS_OK;
   }
+
+  public static String expireStr(
+      DistkvClient distkvClient, DistkvParsedResult parsedResult) {
+    try {
+      DistkvRequest request = parsedResult.getRequest();
+      ExpireRequest expireRequest = request.getRequest().unpack(ExpireRequest.class);
+      distkvClient.strs().expire(request.getKey(), expireRequest.getExpireTime());
```

Comment:
the code  can be more simpler? the only different is `distkvClient.strs()`,  how do you think